config/application.rb

```diff
@@ -44,5 +44,9 @@ class Application < Rails::Application
 
     # Configures the custom queue adapter.
     config.active_job.queue_adapter = :pubsub
+
+    def topic_name
+      ENV["TOPIC_NAME"] || "topic-1"
+    end
```

Comment:
Could be moved to Rails config. I'd also recommend not to set override, but to set defaults:

```rb
Rails.config.active_job.queue_name = "topic-1"
```

Reply:
Done. Addressed on commit: https://github.com/skysemilla/kisi-api-challenge/pull/1/commits/8425060cc4cd4472de51ee68f8080f3ed8d6d764